spec/features/vote_tests/user_votes_on_review_spec.rb

```diff
@@ -0,0 +1,89 @@
+require "rails_helper"
+
+feature "user upvotes a food truck review", %{
+  As an authenticated user
+  I want to upvote a food truck review
+  So that I can give implicit approval
+} do
+  # Acceptance Criteria
+  # If a user is logged in, they can edit vote on reviews.
+
+  scenario "user is NOT logged in and tries voting on a review", js: true do
+    food_truck_review = FactoryGirl.create(:review)
+
+    visit food_truck_path(food_truck_review.food_truck)
+
+    page.find("#novoteforyou").click
+
+    expect(page).to have_content("Log in")
+    expect(page).to have_content("Forgot your password?")
+    expect(page).to_not have_content("Score: 1")
+  end
+
+  scenario "user is logged in, can see a review to vote on", js: true do
+    user = FactoryGirl.create(:user)
+    food_truck_review = FactoryGirl.create(:review, user: user)
+
+    visit new_user_session_path
+
+    fill_in "Email", with: user.email
+    fill_in "Password", with: user.password
+    click_button "Log in"
```

Comment:
Since you're doing this over and over again in this set of tests, it might make sense to use a `before(:each) do`.